.circleci/config.yml

```diff
@@ -9,10 +9,17 @@ jobs:
       - PYVER: "27"
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - build_python_27-{{ .Revision }}
```

Comment:
Would like to make this `{{ .Environment.CIRCLE_JOB }}-{{ .Revision }}`, but for some reason that causes a [syntax error]( https://circleci.com/gh/jakirkham/miniforge/560 ).